### PR TITLE
Init/aws deployment

### PR DESCRIPTION
**Pull request by SEE-DS**

### CODE REVIEW

**Comment by SEE-DS** [comment]
creates one city endpoint; returns city entry by city_id

**Review by axefx — COMMENTED** [review]
looks good thanks. should we gather these csv files into a data folder?
should we remove commented code before merging into main branch or is it still work in progress?

File: project/app/api/city.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    JSON string containing 
+    - **city**: [str] . . . standard format city name, state name
+    - **pop**: [int] . . . population estimate
+    - **age**: [float] . . . average age of residents
+    - **income_household**: [int] . . . median household income
+    - **income_individual**: [int] . . . median individual income
+    - **home**: [int] . . . median home/condo price
+    - **COLI**: [float] . . . Cost of Living Index [ACCRA Cost of Living Index](https://en.wikipedia.org/wiki/ACCRA_Cost_of_Living_Index)
+    """
+
```
Comment (ThomasMcDaniel91) [inline project/app/api/city.py:25]: Great doc string for this.

**Review by ThomasMcDaniel91 — APPROVED** [review]
nice csv, code for endpoint and doc string is very nice.

**Review by LuisFelipeUrena — APPROVED** [review]

**Pull request merged**